### PR TITLE
Feat/message cleanup2 into Develop

### PR DESCRIPTION
**Pull request by pieryvdl**
1. Cleaned up the messaging. The icon has been changed, messaging text has been made bolder, removed green border around profile images, various UI fixes. The icon now only show when typing in the textbox on the chat. You can now press enter/checkmark on keyboard to also send message.  
2. Created a custom backbutton appbar for all the appbars that have backbuttons and changed those appbars to look uniform. 

I can create another custom appbar for the Feed and the profile page which takes actions? ie they both have an icon: Feed has messaging and Profile has settings. Should i do that? 😄

### CODE REVIEW

**Comment by LeratoKgomo** [comment]
the changes on the messages look great 

the custom app bar look great too  , happy from my side

yes the custom appbar for the Feed and the profile pag is a good idea but can we do it later, Im doing major changes on the profile page code including the appbar  once thats finalized then i think we can do this

**Review by LeratoKgomo — APPROVED** [review]

File: lib/view/components/messaging/chat_bubble_component.dart [hunk 1]
```diff
@@ -32,15 +32,16 @@ class ChatBubble extends StatelessWidget {
             message,
             style: TextStyle(
               fontSize: 20,
-              color: Theme.of(context).colorScheme.surface,
-              fontWeight: FontWeight.bold
+              color: Theme.of(context).colorScheme.onSurface,
```
Comment (MonikaTheiss) [inline lib/view/components/messaging/chat_bubble_component.dart:35]: Just a question, what is the difference between onSurface and surface?

**Review by MonikaTheiss — APPROVED** [review]
Looks awesome, thank you! If you want to make the appbars for the feed and profile page, you can, but I don't see a necessity for it currently. If you feel you want to add it, go ahead.
Code also looks good. I did add a small questions with the theme colors, but its nothing major, I was just curious.

File: lib/view/components/messaging/chat_bubble_component.dart (hunk 1 above)
Comment (pieryvdl) [reply lib/view/components/messaging/chat_bubble_component.dart:35]: So surface uses the surface color. Which in this case is white, but that would make the text illegible. So onSurface makes sure to use a color that is legible on the surface it is on. So it uses black for the text in light mode and in dark mode too.

**Pull request merged**